deploy/roles/common/defaults/main.yml

```diff
@@ -3,7 +3,7 @@
 gluster_mount_options: "defaults,acl,_netdev"
 shared_links: []
 
-skip_glusterfs: false
+skip_glusterfs: true
```

Comment:
Could you explain this a bit?  It's probably fine, but I want to make sure that we're not skipping GlusterFS when we don't need to.

Reply:
I'll look into this.

Reply:
I accidentally committed that, and have now changed it back to `false`.  (It didn't affect any of the Travis build's jobs, since they all set `skip_glusterfs` to `true`, except for Job 3, which doesn't use GlusterFS at all anyway.)